lib/tools/adb-commands.js

```diff
@@ -250,10 +251,10 @@ methods.setDataState = async function (on) {
 methods.setWifiAndData = async function ({wifi:w, data:d}) {
```

Comment:
interesting use of ES6 destructured assignment in a parameter list! can we make it conform to our style though and do:

```js
{wifi: w, data: d}
```

Though IMO it would look even more elegant as simply `{wifi, data}`